<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="app_icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="app_name">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Processing...</string>
    </property>
   </widget>
   <widget class="QLabel" name="app_author">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>95</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>processing...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="install">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="vers_lab">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Version :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Link :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="app_link">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="app_vers">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="installingPB">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>541</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="installing_lab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>215</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inria Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Installing</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
